test/fuzz/fuzzer.ml

```diff
@@ -4,10 +4,10 @@ let timeout_count = ref 0
 
 let global_count = ref 0
 
-let write_module filename m = 
-  let oc = open_out filename in
-  Fmt.pf (Format.formatter_of_out_channel oc) "%a@." Owi.Text.pp_modul m;
-  close_out oc
+let write_module (fpath : Fpath.t) m = 
+  match Bos.OS.File.writef fpath "%a@." Owi.Text.pp_modul m with
+  | Ok () -> Ok ()
+  | Error (`Msg err) -> Error (`Msg (Fmt.str "Failed to write module to %a: %s" Fpath.pp fpath err))
```

Comment:
I think this can be simplified as: `let write_module fpath m = Bos.OS.File.writef fpath "%a@." Owi.Text.pp_modul m`.